controller/integration/wrapper/configmap/wrapper.go

```diff
@@ -86,14 +77,17 @@ func New(config Config) (*Wrapper, error) {
 	var newController *controller.Controller
 	{
 		c := controller.Config{
-			Informer: newInformer,
-			Logger:   newLogger,
+			K8sClient: k8sClient,
+			Logger:    newLogger,
 			ResourceSets: []*controller.ResourceSet{
 				resourceSet,
 			},
-			RESTClient: k8sClient.CoreV1().RESTClient(),
+			RuntimeObjectFactory: func() pkgruntime.Object {
+				return &corev1.ConfigMap{}
```

Comment:
```suggestion
				return new(corev1.ConfigMap)
```
Same thing, but reads a bit nicer. Just an idea, I'm fine with both.